openvdb_houdini/houdini/SOP_OpenVDB_Points_Convert.cc

```diff
@@ -1189,27 +1209,29 @@ SOP_OpenVDB_Points_Convert::cookMySop(OP_Context& context)
 
         const fpreal time = context.getTime();
 
-        if (evalInt("conversion", 0, time) != 0) {
+        const int conversion = static_cast<int>(evalInt("conversion", 0, time));
```

Comment:
might be worth adding an enum for this now

Reply:
Done, I had been meaning to add that.